functions/icons.zsh

```diff
@@ -86,6 +87,76 @@ case $POWERLEVEL9K_MODE in
   'awesome-fontconfig')
     # fontconfig with awesome-font required! See
     # https://github.com/gabrielelana/awesome-terminal-fonts
+    # if not defined, set recommended linux path
+    typeset -p "POWERLEVEL9K_FONTAWESOME_PATH" > /dev/null 2>&1  || POWERLEVEL9K_FONTAWESOME_PATH=~/.fonts
```

Comment:
We have a lot of OSX users, and I'm concerned this will break things for them. However, looking at the "Awesome Font" installation instructions for OSX, it doesn't talk about installing these scripts, anywhere.

We either need some OSX users to weigh-in, here, or even better if we could get @gabrielelana's thoughts on how to handle this for OSX, that would be even better.